publish-sdk

```diff
@@ -51,64 +59,70 @@ update_image() {
   local remote_host_arch_image="${remote_image}-${SHORT_SHA}-${host_arch}"
   local remote_alt_arch_image="${remote_image}-${SHORT_SHA}-${alt_arch}"
 
-  if ! docker image inspect "${local_host_arch_image}" >/dev/null 2>&1 ; then
-    echo "did not find ${local_host_arch_image}, skipping it ..." >&2
-    return
-  fi
-
-  if ! docker tag "${local_host_arch_image}" "${remote_host_arch_image}" ; then
-    echo "failed to tag ${remote_host_arch_image}" >&2 && exit 1
+  if [[ "${ONLY_MANIFEST:-"false"}" != "true" ]]; then
```

Comment:
Might be good to add the `u` option to error on unset variables, then default both of the new ones to "false" at the top of the script.

Reply:
I had considered that, but I wanted to allow for **ONLY_MANIFEST** and **SKIP_MANIFEST** to be manipulated with environment variables. It's useful for automated build systems like AWS CodeBuild. I could probably still add the `u` option and just set `ONLY_MANIFEST="${ONLY_MANIFEST:-"false"}"` at the top for the best of both worlds.

Reply:
It looks like we already set `u` within the update image function so we can have the nicer required arguments error. I'll still go ahead and set those two variables at the top though.